oracle/oci-grafana-metrics
# Replaced toml with go mod

- Enables continuous support of plugin support in 7.x  
- Building files are simpler now 
- Build with any golang version > 1.14 

How to test this ?

- Go to the GOPATH (find it using `go env`) 
- create a folder src if not present
- clone this into src 
-  Then do the build formalities `go mod vendor  && yarn && ./build.sh`
- copy to the plugins location and make sure grafana version < 8 

Testing the data source
![image](https://user-images.githubusercontent.com/20325788/128431951-e50e2df5-f4ce-4f03-9e3c-406abcf03b96.png)

Testing the metrics  plugin requests 
![image](https://user-images.githubusercontent.com/20325788/128432097-748b74dd-cb5e-4255-8d7f-dc1ceda2108a.png)



Simpler package management  :)
